src/i2c/master.rs

```diff
@@ -31,6 +31,22 @@ pub enum Speed {
     High,
 }
 
+/// Master SCL Low time. Specifies the minimum low time in clock cycles that will be asserted by this master on SCL.
+#[cfg(feature = "mimxrt685s")]
+pub type Mstscllow = mimxrt685s_pac::i2c0::msttime::Mstscllow;
```

Comment:
`pub use mimxrt685s_pac::i2c0::msttime::Mstscllow;` will work the same way, I think